sklearn/datasets/tests/test_20news.py

```diff
@@ -38,6 +38,19 @@ def test_20news():
     entry2 = data.data[np.where(data.target == label)[0][0]]
     assert_equal(entry1, entry2)
 
+def test_20news_length():
+    """Checks the lenght consistencies within the bunch"""
```

Comment:
typo: length

Please add a comment to mention that this is a non-regression test for a bug in 0.16.1.